code/__DEFINES/status_effects.dm

```diff
@@ -38,3 +38,9 @@
 #define STASIS_CHEMICAL_EFFECT "stasis_chemical"
 
 #define STASIS_ASCENSION_EFFECT "heretic_ascension"
+
+// Stasis helpers
+
+#define STATUS_EFFECT_STASIS_MOBILE /datum/status_effect/grouped/stasis_mobile //SKYRAT ADD: WALKING STASIS
+
+#define IS_IN_STASIS(mob) (mob.has_status_effect(STATUS_EFFECT_STASIS) || mob.has_status_effect(STATUS_EFFECT_STASIS_MOBILE)) //SKYRAT EDIT old: #define IS_IN_STASIS(mob) (mob.has_status_effect(STATUS_EFFECT_STASIS))
```

Comment:
Why is this made here and not where that define is originally at? Unless GitHub doesn't show it to me properly on mobile, this seems like just a pure addition.

Which at that point it could be put in the ~skyrat folder of the __DEFINES folder, so it's modular.